go.mod

```diff
@@ -1,15 +1,30 @@
 module github.com/securesign/rhtas-console
 
-go 1.22.4
+go 1.24
+
+toolchain go1.24.4
```

Comment:
Its important to keep in mind here that there is a chance that when we go to containerize this or release it, we may not have a builder image for go1.24, you can keep an eye on [this](https://catalog.redhat.com/software/containers/rhel9/go-toolset/61df08166d9a1b7b2aab2344) go image its currently at go1.23.9 so we may be  ok, but just something to keep in mind